spec/overlap/ht_item_overlap_spec.rb

```diff
@@ -46,16 +46,33 @@
 
   # we end up inserting the same kinds of things...
   # this adds the given item and holdings to cluster c
-  def cluster_item_holdings(item:, holdings:)
```

Comment:
@mwarin we'd had a TODO here to use `load_test_data` so I went ahead and to-did it